q2_longitudinal/plugin_setup.py

```diff
@@ -221,3 +232,68 @@
         'composition. For more details and citation, please see '
         'doi.org/10.1002/gepi.22065')
 )
+
+
+plugin.methods.register_function(
+    function=first_differences,
+    inputs={'table': FeatureTable[RelativeFrequency]},
+    parameters={**miscellaneous_parameters,
+                **shared_parameters,
+                'metric': Str},
+    outputs=[('first_differences', SampleData[AlphaDiversity])],
+    input_descriptions={
+        'table': ('Feature table to optionally use for computing first '
+                  'differences.')},
+    parameter_descriptions={
+        **miscellaneous_parameter_descriptions,
+        **shared_parameter_descriptions,
+        'metric': ('Numerical metadata or artifact column to test. If '
+                   'computing first differences on a distance matrix, do not '
```

Comment:
outdated docs now that distance matrix support is in its own method

Reply:
👍